Jenkinsfile

```diff
@@ -102,12 +102,25 @@ def buildImage(params, String suffix) {
         }
     }
     // No need for MFS_ROOT kernels when running the testsuite
-    if (!GlobalVars.isTestSuiteJob && suffix.startsWith('riscv64')) {
+    if (!GlobalVars.isTestSuiteJob && (suffix.startsWith('aarch64') || suffix.startsWith('morello') || suffix.startsWith('riscv64'))) {
```

Comment:
Should this be changed to !amd64? Or maybe we can even build them for amd64 as well?

Reply:
We don't have configs for amd64

Reply:
Since it's "you must have explicitly added kernel configs" I went for opt-in not opt-out